packages/workbox-background-sync/Queue.mjs

```diff
@@ -168,7 +170,9 @@ class Queue {
   _addSyncListener() {
     if ('sync' in registration) {
       self.addEventListener('sync', (event) => {
-        event.waitUntil(this.replayRequests());
+        if (event.tag === `${TAG_PREFIX}:${this._name}`) {
+          event.waitUntil(this.replayRequests());
```

Comment:
Can you add a log here to signal the sync event was received and replaying is being started.

Reply:
Done